modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/JacksonGenerator.scala

```diff
@@ -429,17 +434,22 @@ object JacksonGenerator {
           terms.filterNot(term => discriminatorNames.contains(term.propertyName))
 
         terms.foreach({
-          case ParameterTerm(propertyName, parameterName, fieldType, _, _, _, _) =>
+          case ParameterTerm(propertyName, parameterName, fieldType, _, _, _, _, constraints) =>
             val field: FieldDeclaration = dtoClass.addField(fieldType, parameterName, PRIVATE, FINAL)
             field.addSingleMemberAnnotation("JsonProperty", new StringLiteralExpr(propertyName))
+            for (constraint <- constraints) yield (constraint) match {
+              case Minimum(value) => field.addSingleMemberAnnotation("Min", new IntegerLiteralExpr(value.toString))
+              case Maximum(value) => field.addSingleMemberAnnotation("Max", new IntegerLiteralExpr(value.toString))
```

Comment:
Thoughts on just using the fully qualified class name here instead of prematurely adding it to the imports?

Reply:
I'm torn on this -- fully qualified is nice to prevent conflicts with names that end up getting generated from the spec file (like maybe there's a DTO called `Max` in `components/schemas`), but it also makes the generated code much harder to read.  I think the scala generators have been pushing in the direction of FQNs, but for java I haven't gone that way yet.  So for now I think this is fine.